client/src/pages/Login.js

```diff
@@ -44,61 +39,67 @@ export default function Login() {
   const [password, setPassword] = useState('');
 
   const handleClick = () => {
-    axios.post(`${baseURL}/user/login`, {
-      email,
-      password
-    }).then(response => {
-      history.push('/messenger');
-    }).catch(error => console.error(error))
+    axios
+      .post(`/user/login`, {
+        email,
+        password,
+      })
+      .then((response) => {
```

Comment:
We want to check the success of the response first, and also move the axios call into userContext